src/App.tsx

```diff
@@ -1,61 +1,136 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+import { FormEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+export const todosWithUsers = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
```

Comment:
[REACT KNOWLEDGE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. It's a good practice to prepare all the data in the root component and pass it to child components as props. In this case, you are preparing `todosWithUsers` in the global scope of this module. It would be better to move this logic inside the `App` component.